src/Schema/Config.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+namespace StellarWP\Schema;
+
+class Config {
+	/**
+	 * Container object.
+	 *
+	 * @var object
```

Comment:
Going with the PSR-11 standard would mean this dependency type-hint would be `Psr\Container\ContainerInterface` from the [psr/container](https://packagist.org/packages/psr/container) Composer package.